### PR TITLE
Chore(cirriculum) - Use alternate wording for step 36 of the cafe menu.

### LINKED ISSUE #47958: Make Cafe Menu Step 36 instructions a little clearer
### Describe the Issue

Most of the description for step 37 is about styling `p` elements. It even states that "you need to apply some styling to the `p` elements". But then suddenly the last line instructs the user to add a `class` to an `article`, which seems a little confusing and out of place. This caused a least one camper to add a new `p` element in this step and apply the class attribute to that element. 

### Affected Page

~~https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-37~~

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-36

### Your code

Campers code:

```
<article>
            <p class="item"></p>
            <p class="flavor">French Vanilla</p>
            <p class="price">3.00</p>
</article>
```

### Expected behavior

Even though the last line is clear that the class should be added to the article, I think it could be made less confusing if we explained that this is in preparation for styling the `p` elements. Perhaps change the last sentence to something like:

> To help with styling the `p` elements, you will first add a `class` attribute with the value `item` to the first `article` element under the `Coffee` heading.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #47958

### CODE REVIEW

**Comment by a2937** [comment]
Hmm should this be closed along with the associated issue as as the wording probably already was satisfactory?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866de7a5b784048f94b1.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: step-36
 
 That is kind of what you want, but now it would be nice if the flavor and price were on the same line. `p` elements are <dfn>block-level</dfn> elements, so they take up the entire width of their parent element.
 
-To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. Add a `class` attribute with the value `item` to the first `article` element under the `Coffee` heading.
+To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. To do that, start by adding a `class` attribute with the value `item` to their parent `article` element.
```
Comment (bbsmooth) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866de7a5b784048f94b1.md:12]:
```suggestion
To get them on the same line, you need to apply some styling to the `p` elements, so they behave more like <dfn>inline</dfn> elements. To do that, start by adding a `class` attribute with the value `item` to the first  `article` element under the `Coffee` heading.
```

I don't mind the addition of `To do that, start by adding` but I don't think the last part of that sentence should be changed.

**Comment by bbsmooth** [comment]
The wording on step 36 is probably good enough as it is, so this PR is not essential. But I do think the addition of `To do that, start by...` (or something similar) makes the transition from explanation to instruction a little nicer.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866de7a5b784048f94b1.md (hunk 1 above)
Comment (a2937) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f3c866de7a5b784048f94b1.md:12]: Understood.

**Review by Sembauke — APPROVED** [review]

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**